README.adoc

```diff
@@ -300,77 +297,77 @@ AppDeploymentConfig.java
+Annotate the `PersonServiceIT` class with the `@SharedContainerConfiguration` annotation that references the `AppDeploymentConfig` shared configuration class.
 [role="code_command hotspot file=3", subs="quotes"]
 ----
 #Update the `PersonServiceIT` class.#
 `src/test/java/io/openliberty/guides/testing/PersonServiceIT.java`
 ----
 [role="edit_command_text"]
-Import [hotspot=importSharedContainerConfig file=3]`SharedContainerConfig` annotation, and annotate the `PersonServiceIT` class with [hotspot=sharedContainerConfig file=3]`@SharedContainerConfig`. 
+Import the [hotspot=importSharedContainerConfig file=3]`SharedContainerConfig` annotation and annotate the `PersonServiceIT` class with [hotspot=sharedContainerConfig file=3]`@SharedContainerConfig`. 
 
 PersonServiceIT.java
 [source, java, linenums, role='code_column hide_tags=copyright']
 ----
 include::finish/src/test/java/io/openliberty/guides/testing/PersonServiceIT.java[]
 ----
 
-Then, update the `ErrorPathIT` class similarly to remove the container code.
+Similarly, update the `ErrorPathIT` class to remove the container code.
 [role="code_command hotspot file=4", subs="quotes"]
 ----
 #Update the `ErrorPathIT` class.#
 `src/test/java/io/openliberty/guides/testing/ErrorPathIT.java`
 ----
 [role="edit_command_text"]
-Remove [hotspot=importMPApp hotspot=importContainer file=4]`import` statements and [hotspot=container hotspot=mpApp file=4]`MicroProfileApplication app` field
+Remove [hotspot=importMPApp hotspot=importContainer file=4]`import` statements and the [hotspot=container hotspot=mpApp file=4]`MicroProfileApplication app` field
 
 ErrorPathIT.java
 [source, java, linenums, role='code_column hide_tags=copyright']
 ----
 include::hotspots/src/test/java/io/openliberty/guides/testing/ErrorPathIT.java[]
 ----
 
-Annotate the `ErrorPathIT` class with `@SharedContainerConfiguration`.
+Annotate the `ErrorPathIT` class with the `@SharedContainerConfiguration` annotation.
 [role="code_command hotspot file=5", subs="quotes"]
 ----
 #Update the `ErrorPathIT` class.#
 `src/test/java/io/openliberty/guides/testing/ErrorPathIT.java`
 ----
 [role="edit_command_text"]
-Import [hotspot=importSharedContainerConfig file=5]`SharedContainerConfig` annotation, and annotate the `ErrorPathIT` class with [hotspot=sharedContainerConfig file=5]`@SharedContainerConfig`. 
+Import the [hotspot=importSharedContainerConfig file=5]`SharedContainerConfig` annotation and annotate the `ErrorPathIT` class with [hotspot=sharedContainerConfig file=5]`@SharedContainerConfig`. 
 
 ErrorPathIT.java
 [source, java, linenums, role='code_column hide_tags=copyright']
 ----
 include::finish/src/test/java/io/openliberty/guides/testing/ErrorPathIT.java[]
 ----
 
-Now, if you rerun the tests, they will run in about half the time because the same server instance is being used for both test classes.
+If you rerun the tests now, they run in about half the time because the same server instance is being used for both test classes.
```

Comment:
Add a colon at the end of **If you rerun the tests now, they run in about half the time because the same server instance is being used for both test classes**, so that it leads into the Maven goal.

Reply:
addressed